deploy/examples/create-external-cluster-resources.py

```diff
@@ -1364,21 +1383,28 @@ def _gen_output_map(self):
         self.out_map[
             "RBD_METADATA_EC_POOL_NAME"
         ] = self.validate_rbd_metadata_ec_pool_name()
-        self.out_map["RGW_REALM_NAME"] = self._arg_parser.rgw_realm_name
         self.out_map["RGW_POOL_PREFIX"] = self._arg_parser.rgw_pool_prefix
         self.out_map["RGW_ENDPOINT"] = ""
         if self._arg_parser.rgw_endpoint:
             if self._arg_parser.dry_run:
                 self.create_rgw_admin_ops_user()
             else:
+                err = self.validate_rgw_multisite(
+                    self._arg_parser.rgw_realm_name, "--rgw-realm"
+                )
+                err = self.validate_rgw_multisite(
+                    self._arg_parser.rgw_zonegroup_name, "--rgw-zonegroup"
+                )
+                err = self.validate_rgw_multisite(
+                    self._arg_parser.rgw_zone_name, "--rgw-zone"
+                )
                 (
                     self.out_map["RGW_ADMIN_OPS_USER_ACCESS_KEY"],
                     self.out_map["RGW_ADMIN_OPS_USER_SECRET_KEY"],
                     info_cap_supported,
                     err,
                 ) = self.create_rgw_admin_ops_user()
```

Comment:
Sorry I don't understand how we are checking whether adminsop user exists or not? Is the check already present in the script?

Reply:
https://github.com/rook/rook/blob/85d7920b2ae9180594375da06d2c159d6087c5d2/deploy/examples/create-external-cluster-resources.py#L1083

Reply:
Thanks for confirming